bin/stress-test/src/seeding/mod.rs

```diff
@@ -396,6 +404,11 @@ fn create_consume_note_tx(
         Word::empty(),
         block_ref.block_num(),
         block_ref.commitment(),
+        FungibleAsset::new(
+            block_ref.fee_parameters().native_asset_id(),
+            u64::from(block_ref.fee_parameters().verification_base_fee()),
+        )
```

Comment:
This is probably fine for now, but it may be better to have a helper function to create this fee (similar to how we have `test_fee()` functions elsewhere). This function could take `&FeeParameters` as an input and output `FungibleAsset`.

Reply:
I added the helper, if we want, we can move that into the `FeeParameters`